src/components/MenuBar/MenuBar.scss

```
@@ -0,0 +1,100 @@
+.menuBarFrame {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 279px;
+  height: 100vh;
```

페이지 내용이 100vh가 넘어가게 된다면, 이 menu bar는 100vh에서 잘려서 보이겠네요.
아마 이런 식으로 넣어주면 될 것 같습니다.
```suggestion
  height: 100%;
  min-height: 100vh;
```

---

아까 알려주신 Sticky 사용해서 반영하였습니다.